src/app/modules/group/pages/group-user/group-user.component.html

```diff
@@ -1,37 +1,28 @@
 <ng-container *ngIf="state$ | async as state">
-  <div *ngIf="state.isFetching">
-    Loading...
-  </div>
 
-  <div *ngIf="state.isError">
-    Error
-  </div>
+  <p-progressSpinner *ngIf="state.isFetching"></p-progressSpinner>
+
+  <p *ngIf="state.isError" i18n>Error while loading the user info</p>
 
   <ng-container *ngIf="state.isReady && state.data as user">
 
     <div class="alg-page-header">
       <div class="alg-page-header-title">
-        <span *ngIf="user.firstName || user.lastName; else login">
+        <span *ngIf="user.firstName && user.lastName; else login">
           {{ user.firstName }} {{ user.lastName }}
```

Comment:
Still not. It is supposed to show  "first_name last_name (login)" (so the login between brackets) when  first_name/last_name are known.

Reply:
Fixed